app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -42,10 +77,78 @@ object FirefoxTheme {
+    layer2 = PhotonColors.Ink50,
+    layer3 = PhotonColors.Ink50,
+    layer4Start = PhotonColors.Purple70,
+    layer4Center = PhotonColors.Violet80,
+    layer4End = PhotonColors.Ink05,
+    layerAccent = PhotonColors.Violet40,
+    layerAccentNonOpaque = PhotonColors.Violet50A32,
+    layerAccentOpaque = Color(0xFF423262),
+    scrim = PhotonColors.DarkGrey90A95,
+    gradientStart = PhotonColors.Violet70,
+    gradientEnd = PhotonColors.Violet40,
+    actionPrimary = PhotonColors.Violet60,
+    actionSecondary = PhotonColors.LightGrey05,
+    actionTertiary = PhotonColors.DarkGrey10,
+    actionQuarternary = PhotonColors.DarkGrey80,
+    formDefault = PhotonColors.LightGrey05,
+    formSelected = PhotonColors.Violet40,
+    formSurface = PhotonColors.DarkGrey05,
+    formDisabled = PhotonColors.DarkGrey05,
+    formOn = PhotonColors.Violet40,
+    formOff = PhotonColors.LightGrey05,
+    indicatorActive = PhotonColors.LightGrey90,
+    indicatorInactive = PhotonColors.DarkGrey05,
+    textPrimary = PhotonColors.LightGrey05,
+    textSecondary = PhotonColors.LightGrey40,
+    textDisabled = PhotonColors.LightGrey05A40,
+    textWarning = PhotonColors.Red20,
+    textWarningButton = PhotonColors.Red70,
+    textAccent = PhotonColors.Violet20,
+    textAccentDisabled = PhotonColors.Violet20A60,
+    textOnColorPrimary = PhotonColors.LightGrey05,
+    textOnColorSecondary = PhotonColors.LightGrey40,
+    textActionPrimary = PhotonColors.LightGrey05,
+    textActionSecondary = PhotonColors.DarkGrey90,
+    textActionTertiary = PhotonColors.LightGrey05,
+    textActionTertiaryActive = PhotonColors.LightGrey05,
+    iconPrimary = PhotonColors.LightGrey05,
+    iconPrimaryInactive = PhotonColors.LightGrey05A60,
+    iconSecondary = PhotonColors.LightGrey40,
+    iconActive = PhotonColors.Violet40,
+    iconDisabled = PhotonColors.LightGrey05A40,
+    iconOnColor = PhotonColors.LightGrey05,
+    iconNotice = PhotonColors.Blue30,
+    iconButton = PhotonColors.LightGrey05,
+    iconWarning = PhotonColors.Red20,
+    iconWarningButton = PhotonColors.Red70,
+    iconAccentViolet = PhotonColors.Violet20,
+    iconAccentBlue = PhotonColors.Blue20,
+    iconAccentPink = PhotonColors.Pink20,
+    iconAccentGreen = PhotonColors.Green20,
+    iconAccentYellow = PhotonColors.Yellow20,
+    iconActionPrimary = PhotonColors.LightGrey05,
+    iconActionSecondary = PhotonColors.DarkGrey90,
+    iconGradientStart = PhotonColors.Violet20,
+    iconGradientEnd = PhotonColors.Blue20,
+    borderPrimary = PhotonColors.DarkGrey05,
+    borderInverted = PhotonColors.LightGrey30,
+    borderFormDefault = PhotonColors.LightGrey05,
+    borderAccent = PhotonColors.Violet40,
+    borderDisabled = PhotonColors.LightGrey05A40,
+    borderWarning = PhotonColors.Red40
+)
+
 private val darkColorPalette = FirefoxColors(
     layer1 = PhotonColors.DarkGrey60,
     layer2 = PhotonColors.DarkGrey30,
     layer3 = PhotonColors.DarkGrey80,
+    layer4Start = PhotonColors.Purple70,
```

Comment:
just to confirm, are these new layer colors supposed to be the same for all 3 palettes?

Reply:
It's undefined for Light and Dark theme since it's technically private only.